app/src/main/java/com/sopt/now/compose/ui/home/Home.kt

```diff
@@ -32,15 +35,39 @@ import androidx.compose.ui.unit.sp
 import androidx.navigation.NavHostController
 import androidx.navigation.compose.rememberNavController
 import com.sopt.now.compose.R
+import com.sopt.now.compose.component.UiState
 import com.sopt.now.compose.component.bottomnavigation.BottomNavigation
 import com.sopt.now.compose.component.bottomnavigation.BottomNavigationItem
 import com.sopt.now.compose.component.text.TextWithTitle
+import com.sopt.now.compose.component.toastMessage
 import com.sopt.now.data.model.User
 import com.sopt.now.data.model.UserViewModel
 
 @Composable
 fun Home(navHostController: NavHostController, viewModel: UserViewModel) {
```

Comment:
```suggestion
fun Home(navHostController: NavHostController, viewModel: UserViewModel = viewModel()) {
```

컴포즈에서는 이렇게 사용하는 경우도 많은 것 같더라구요!

Reply:
오 이렇게하면 preview에 뷰모델을 전달 안해도 되는군요! 감사합니다.